docs/setup/installation-and-configuration.md

```diff
@@ -0,0 +1,87 @@
+## Installation
+
+The Mattermost MS Teams Meetings plugin is provided in the Mattermost Plugin Marketplace. Once enabled, clicking the video icon in a Mattermost channel invites team members to join an MS Teams meeting, hosted using the credentials of the user who initiated the call.
+
+### Requirements
+
+* Mattermost Server v5.26+ is required.
+* Enterprise Edition E20 is required.
+
+### Marketplace Installation
+
+1. Go to **Main Menu > Plugin Marketplace** in Mattermost.
+2. Search for "MS Teams" or manually find the plugin from the list and click **Install**.
+3. After the plugin has downloaded and been installed, click **Configure**.
+
+### Manual Installation
+
+If your server doesn't have access to the internet, you can download the latest [plugin binary release](https://github.com/mattermost/mattermost-plugin-msteams-meetings/releases) and upload it to your server via **System Console > Plugin Management**. The releases on this page are the same used by the Marketplace.
+
+## Configuration
+
+### Step 1: Create a Mattermost App in Azure
+
+1. Sign into [portal.azure.com](https://portal.azure.com) using an admin Azure account.
```

Comment:
Wonder if it would be better as the name of the link something like "Azure portal". Something like:

Sign into the [Azure portal](xxx) using an admin account.

Reply:
Good catch, I'll update it.